solid/contacts-pane
# Feature: Enable dev mode

This enables you to work standalone with the new auth.

To test: 
1.  Bring down branch 
2.  Change links to webid and address book to your own in index.ts
3. Npm run start

You should then see your address book at localhost:8080

Note: This is now in feature styles branch.  I will close it off if that is accepted otherwise I will come back to this and change the few things that need to change after auth refactor.
